css/main.css

```diff
@@ -1,3 +1,99 @@
 /* Add your custom CSS here */
 
+
+.hero {
+    height: 500px;
+    background-image: url(../images/rockclimbing.jpg);
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+} 
+
+header {
+    background-color: rgb(191, 211, 248);
+    width: 100%;
+
+}
+
+.ambition {
+    font-size: 2.5vw;
+}
+
+.success {
+    font-size: 4vw;
+    font-weight: 200;
+    margin: 0;
+}
+
+ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.book-now {
+    text-align: center;
+    font-size: 1.5vw;
+    background-color: black;
+    color: white;
+}
+
+.row3 .col-6 {
+    height: 400px;
+}
+
+.row3 .left {
```

Comment:
In stead of overriding these grid classes, you could create some new classes that are more specific to this style. e.g. `.light-blue`.

This will make it easier for people in the future to know how this works.